docs/en/ingest-management/faq.asciidoc

```diff
@@ -152,11 +152,45 @@ downloads from the {package-registry} at `epr.elastic.co`. This means that
 
 [discrete]
 [[what-is-the-endpoint-package]]
-== What is the {elastic-endpoint-integration} integration in {ingest-manager}?
+== What is the {elastic-endpoint-integration} integration in {fleet}?
 
 The {elastic-endpoint-integration} integration provides protection on your {agent}
 controlled host. The integration monitors your host for security-related events,
 allowing for investigation of security data through the {security-app} in {kib}.
 The {elastic-endpoint-integration} integration is managed by {agent} in the
 same way as other integrations. Try it out! For more information, see the
 {security-guide}/index.html[{elastic-sec} solution documentation].
+
+[discrete]
+[[how-are-security-to-agent-communications-secured]]
+== How are communications secured between {elastic-sec} and {agent}?
+
+{elastic-sec} connects to the agent over loopback TLS on port 6788.
+{elastic-sec} validates that the agent has root (Linux and macOS) or SYSTEM
+(Windows) permissions.
+
+//REVIEWERS: Unsure here whether we want to say Elastic Endpoint or Elastic
+//Security. ^^
+
+//REVIEWERS: In 7.10, does Elastic Security validate the digital signature of
```

Comment:
Hey, @ferullo - do you know the answer to this question?

Reply:
It does not in 7.10.

Reply:
In 7.10 Agent validates the signature of Endpoint before installing Endpoint. I don't think it validates it after that.